File: exercicios em Python/q43.py
import os

print("\n\nEste programa determina se a media e a soma de n numero e impar ou par.")
n = int(input("\nDigite a quantidade de numeros a qual deseja digitar: "))
soma = 0
i = 1
while i <= n:
        print("\n\nDigite o ",i," numero: ")
        num = int(input(""))
        soma = int(soma + num)
        i = i + 1
        
media = int(soma / n)
if soma % 2 == 0:
        print("\nA soma total e ",soma,", um numero par, e a media e")
else:
        print("\nA soma total e ",soma,", um numero impar, e a media e")

if media % 2 == 0:
        print(media," um numero Par.")
else: 
        print(media," um numero Impar.")

print("\nObrigado por utilizar nosso sistema!")
os.system("pause")

File: exercicios em C/q43.cpp
#include <stdio.h>

int main(void){
	int num, media, i, n, soma;
	media = 0;
	soma = 0;
	printf("\n\nEste programa determina se a media e a soma de n numero e impar ou par.");
	printf("\n\nDigite a quantidade de numeros a qual deseja digitar: ");
	scanf("%d", &n);
	for(i = 1; i <= n; i++){
		printf("\n\nDigite o %d numero: ", i);
		scanf("%d", &num);
		soma = soma + num;
	}
	media = soma / n;
	if (soma % 2 == 0){
    	printf ("\nA soma total e %d, um numero par, e a media e",soma);
	}
	else {
    	printf ("\nA soma total e %d, um numero impar, e a media e",soma);
	}
	if (media % 2 == 0){
    	printf (" %d, um numero Par.",media);
	}
	else {
    	printf(" %d, um numero Impar.",media);
	}
	printf("\n\nObrigado por utilizar nosso sistema!");
	return 0;
}
